components/transactions/TransactionDetails.js

```diff
@@ -74,11 +81,21 @@ const TransactionDetails = ({ displayActions, transaction, isRoot, isHostAdmin,
     order,
     expense,
     isRejected,
+    createdAt,
   } = transaction;
-  const userCanRejectOrRefund = isRoot || isHostAdmin;
+  // calculate 30 days ago from transaction
+  const thirtyDaysInMillisecond = 1000 * 60 * 60 * 24 * 30;
```

Comment:
:ice_cream:: It's common to declare pure constants in the global scope using CAPSIZED_SNAKE_CASE. It conveys you're dealing with something immutable during the whole run-time.